backend/routes/logout.ts

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+
+const logoutRouter = express.Router();
+
+
+logoutRouter.post('/logout', function(req, res, next){
+    req.logout(function(err) {
+      if (err) {
+        return next(err);
+      }
+      req.session.destroy(function(err) {
+        if (err) {
+          return next(err);
+        }
+        res.redirect('/login');
```

Comment:
nit: let's make this go to home page!

Reply:
home page meaning the tempLandingPage I added?